### PR TITLE
Update poketex package installation

### PR DESCRIPTION
**Pull request by ckaznable**
Hello!
I have added ansi image files to poketex.
This is my first time editing a PKGBUILD file, so please let me know if you need to make any changes.

Thank you for maintaining this repo!

### CODE REVIEW

File: poketex-git/PKGBUILD [hunk 1]
```diff
@@ -37,6 +37,9 @@ check() {
 
 package() {
   cd "${pkgname%-git}"
+  mkdir -p "$pkgdir/usr/share/${pkgname%-git}/colorscripts/small/"
```
Comment (alerque) [inline poketex-git/PKGBUILD]: Since you have `install -Dt` going on here anyway you don't need two `mkdir -p` calls.

Comment (ckaznable) [reply poketex-git/PKGBUILD]: removed, Thanks!

**Review by alerque — APPROVED** [review]
LGTM. Obviously this is Orhun's package to decide style. If it was mine I would nitpick the readablility by changing the order so the main gig gets installed first (in this case the binary), then any support files, then Arch specific considerations like docs and licenses. But then if it was mine I would but the `install -t` arg before the list of files being installed but obviously Orhun has different preferences there. In any case style nits aside this looks *correct* now.

**Review by orhun — APPROVED** [review]

**Comment by orhun** [comment]
Thanks @ckaznable and thanks @alerque for the review!

**Pull request merged**